stories/components/TextArea/TextArea.stories.tsx

```diff
@@ -31,33 +32,38 @@ const TextAreaStory: React.FC = () => {
           />
           <TextArea label="Name" secondaryLabel="Optional" />
           <TextArea
+            aria-label="Name"
             label="Name"
```

Comment:
Should `aria-label` only be required on the text areas without a `label`? Or does it complain?

Reply:
It's alerting this on every `<TextArea` Any ideas?

<img width="510" alt="Screen Shot 2021-03-18 at 1 13 19 PM" src="https://user-images.githubusercontent.com/485903/111668194-e42ab300-87eb-11eb-9fea-ee35bc7751f5.png">

Reply:
I am wrapping up a PR and then I can take a quick look at why this is happening.

Reply:
I think it may actually be `aria-hidden="true"` - I think we can get rid of that if I'm not mistaken 

https://github.com/lifeomic/chroma-react/blob/feature/color-contrast/src/components/TextArea/TextArea.tsx#L201

Reply:
👀

Reply:
That did the trick, thanks @ynotdraw

Reply:
Any time! Not sure why I added that to begin with, so sorry for the troubles